com.endlessm.HackAvatarFaces.Extension.appdata.xml

```diff
@@ -0,0 +1,24 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="addon">
+  <id>com.endlessm.HackAvatarFaces.extension</id>
+  <extends>com.endlessm.HackComponents</extends>
+  <metadata_license>CC0-1.0</metadata_license>
+  <project_license>LicenseRef-proprietary</project_license>
+  <categories>
+    <category>Game</category>
+  </categories>
```

Comment:
Categories are not needed here, since we're not supposed to show this anywhere in any case.